app/src/main/java/com/xmartlabs/template/MainActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.xmartlabs.template
+
+import android.os.Bundle
+import com.f2prateek.dart.HensonNavigable
+import com.xmartlabs.bigbang.ui.mvp.BaseMvpAppCompatActivity
+import com.xmartlabs.bigbang.ui.mvp.BaseMvpPresenter
+import com.xmartlabs.bigbang.ui.mvp.MvpView
+
+import kotlinx.android.synthetic.main.activity_main.*;
+
+@HensonNavigable
+class MainActivity : BaseMvpAppCompatActivity<MvpView, BaseMvpPresenter<MvpView>>() {
+  override lateinit var presenter: BaseMvpPresenter<MvpView>
```

Comment:
It comes from super, and it must be a `var` to be lateinitiated. And it must be lateinit in order to inject it.